understanding/consistent-help.html

```diff
@@ -53,322 +52,225 @@
                      </p>
                   </div>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>簡潔に</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Make it easier to find help and support.</dd>
+                  <dt>目標</dt>
+                  <dd>ヘルプ及びサポートを見つけやすくする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Put help in the same place when it is on multiple pages.</dd>
+                  <dt>何をするか</dt>
+                  <dd>ヘルプが複数のページにある場合は、同じ場所にヘルプを配置する。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>People who need help can find it more easily if it's in the same place.</dd>
+                  <dt>なぜ重要か</dt>
+                  <dd>同じ場所にあれば、ヘルプを必要とする人は、より簡単に見つけることができる。</dd>
                   
                </dl>
             </section>
             <section id="intent">
-               <h2>Intent</h2>
-               <p>The intent of this Success Criterion is to ensure users can find help for completing
-                  tasks on a Web site, when it is available. When the placement of the help mechanism
-                  is kept consistent across a set of pages, users looking for help will find it easier
-                  to identify. This is distinct from interface-level help, such as contextual help,
-                  features like spell checkers, and instructional text in a form. 
+               <h2>意図</h2>
+               <p>この達成基準の意図は、ウェブサイト上で作業を完了するためのヘルプが利用可能な場合に、そのヘルプを利用者が確実に見つけられるようにすることである。ヘルプメカニズムの配置が一連のページ全体で一貫していると、ヘルプを探している利用者が見つけやすくなる。これは、コンテキストヘルプ、スペルチェッカーのような機能、フォーム内の説明テキストなどのインターフェースレベルのヘルプとは異なる。
                </p>
-               <p>Locating the help mechanism in a consistent location across pages makes it easier
-                  for users to find it.  For example, when a mechanism or link is located in the header
-                  of one Web page, it will be easier to find if it is in the header of other pages.
-                  The help mechanism, such as a contact phone number, may be provided directly on the
-                  page, or it may also be a direct link to a contact page. Regardless of which approach
-                  is used, the mechanism must be located in the same relative order on each page within
-                  the set of pages.
+               <p>ヘルプメカニズムをページ間で一貫した場所に配置すると、利用者が見つけやすくなる。例えば、あるメカニズム又はリンクが一つのウェブページのヘッダーにあるとき、それが他のページのヘッダーにある場合、見つけやすくなる。連絡先の電話番号などのヘルプメカニズムは、ページ上に直接提供されてもよく、又は連絡先ページへの直接リンクであってもよい。どのアプローチを使用するかに関係なく、そのメカニズムは、その一連のページ内の各ページに相対的に同じ順序で配置されなければならない。
```

Comment:
「一連のページ」原文では a set of pages で、おそらく「a set of web pages (ウェブページ一式)」と同義かと思います。

```suggestion
               <p>ヘルプメカニズムをページ間で一貫した場所に配置すると、利用者が見つけやすくなる。例えば、あるメカニズム又はリンクが一つのウェブページのヘッダーにあるとき、それが他のページのヘッダーにある場合、見つけやすくなる。連絡先の電話番号などのヘルプメカニズムは、ページ上に直接提供されてもよく、又は連絡先ページへの直接リンクであってもよい。どのアプローチを使用するかに関係なく、そのメカニズムは、ウェブページ一式の中で各ページに相対的に同じ順序で配置されなければならない。
```

Reply:
ここも、a set of pages は「ウェブページ一式」ではなく、「一連のページ群」として、以下のようにする形で、いかがでしょうか。

```suggestion
               <p>ヘルプメカニズムをページ間で一貫した場所に配置すると、利用者が見つけやすくなる。例えば、あるメカニズム又はリンクが一つのウェブページのヘッダーにあるとき、それが他のページのヘッダーにある場合、見つけやすくなる。連絡先の電話番号などのヘルプメカニズムは、ページ上に直接提供されてもよく、又は連絡先ページへの直接リンクであってもよい。どのアプローチを使用するかに関係なく、そのメカニズムは、その一連のページ群の中の各ページに相対的に同じ順序で配置されなければならない。
```